src/Spout/Writer/Style/BorderBuilder.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Box\Spout\Writer\Style;
+
+/**
+ * Class BorderBuilder
+ */
+class BorderBuilder
+{
+    /**
+     * @var Border
+     */
+    protected $border;
+
+    public function __construct()
+    {
+        $this->border = new Border();
+    }
+
+    /**
+     * @param string|void $style Border style @see BorderPart::allowedStyles
+     * @param string|void $color Border A RGB color code
+     * @param string|void $width Border width @see BorderPart::allowedWidths
+     * @return self
+     */
+    public function setBorderTop($style = Border::STYLE_SOLID, $color = Color::BLACK, $width = Border::WIDTH_THICK)
```

Comment:
I did not realize I asked you twice to re-order these values in different order... My bad!
The order is pretty hard to get right (that's why I try to avoid functions with default values - internal default values that can be overridden via setters is better!). In this case, it's a bit overkill though. Just need to find what the best order is :)